templates/guidelines.html

```diff
@@ -51,24 +51,24 @@ <h2 class="Guidelines__section__head">Moderator System</h2>
                 <p> Calendar moderators maintain the events on the calendar to guarantee that Frank’s Calendar is populated with <span class="Guidelines__underline">accurate, up to date information</span> about what is happening on campus. They also monitor events to make sure they meet the requirements of what is acceptable to share. </p>
                     <p> <span class="Guidelines__underline">Their admin privileges include:</span></p>
                     <ul>
-                        <li>+ Reviewing event submissions</li>
-                        <li>+ Approving, requesting changes, or cancelling an event submission</li>
-                        <li>+ Sharing the link to edit an event if hosts lose their link</li>
+                        <li>Reviewing event submissions</li>
+                        <li>Approving, requesting changes, or cancelling an event submission</li>
+                        <li>Sharing the link to edit an event if hosts lose their link</li>
                     </ul>
                     <p>If you're interested in becoming a moderator or developer for Frank's Calendar, please contact one of the current moderators listed below.</p>
                     <h3 class="Guidelines__mods">Current Moderators</h3>
                     <div class="Guidelines__centered">
-                        <p>Jules Brettle (she/her) | <a href="mailto:jbrettle@olin.edu">jbrettle@olin.edu</a></p>
-                        <p>Raquel Dunoff (she/they) | <a href="mailto:rdunoff@olin.edu">rdunoff@olin.edu</a> </p>
+                        <p>Jules Brettle, <a href="mailto:jbrettle@olin.edu">jbrettle@olin.edu</a></p>
+                        <p>Raquel Dunoff, <a href="mailto:raquel.dunoff@students.olin.edu">raquel.dunoff@students.olin.edu</a></p>
```

Comment:
Shouldn't we keep their pronouns in? And is this the wrong email?

Reply:
yeah I didn't mean to do this. this was an error when merging